lib/wallaby/query.ex

```diff
@@ -54,6 +54,18 @@ defmodule Wallaby.Query do
     end
   end
   ```
+
+  ## What does my query do?
+
+  Wanna check out what exactly your query will do? Look no further than
+  `Wallaby.Query.compile/1` - it takes a query and returns the css or xpath
+  query that will be sent to the driver:
+
+      iex> Wallaby.Query.compile Wallaby.Query.text("my text")
+      {:xpath, ".//*[contains(normalize-space(text()), 'my text')]"}
```

Comment:
Its good to know that you have to indent the doctest instead of using ``` before Elixir 1.4.

Reply:
Yeah remembered that bit, was even surprised the other way works now :D Grew accustomed to the indentation